src/codegen.ts

```diff
@@ -9,7 +9,8 @@ import { generateSqlFnFromAst } from './generateSqlFnFromAst';
 import { generateReturnTypeFromAst } from './generateReturnTypeFromAst';
 import { astify } from './parser';
 import { generateParamsTypeFromAst } from './generateParamsTypeFromAst';
-import { getTableDefinition } from './getTableDefinition';
+// import { getTableDefinition } from './getTableDefinition';
+import { getMultiTableDefinition } from './getTableDefinition';
```

Comment:
I would rename the file to getTablesDefinition having a single method (remove the old one) with the same name

Reply:
Done